crates/index/src/config_json.rs

```diff
@@ -19,17 +19,22 @@ impl ConfigJson {
         protocol: Protocol,
         api_address: &str,
         api_port: u16,
+        api_url_path: &str,
```

Comment:
Would be nice if this is an `Option`. This would mean that we can hand around a `None` instead of `""`, which is more expressive.